Gaijinizer patch/Troops.txt

```diff
@@ -4344,9 +4344,9 @@ by humans ♡#Triketos
 
 悪魔の『契約』が成り立っちゃうから、お触りだけね♡
 > CONTEXT: Troops/90/pages/1/37/Dialogue/enemy5
-Remember what I said before~...
-Because I have to keep my promise, so I can only
-touch you, okay? ♡#Triketos
+But I'm a demon so if we have actual sex I'll
+end up absorbing your energy and forming a
+contract with you, so it's touching only, okay? ♡#Ron
```

Comment:
Flow:
>_**But I'm a demon so if we have actual sex I'll
end up absorbing your energy and forming a
contract with you, so... touching only, okay? ♡**_
(It's a really long sentence, but it's kinda hard to break it up more then this...)